Dockerfile

```diff
@@ -29,3 +29,7 @@ ENV PATH $PATH:/go/bin
 ENV GOPATH=/go
 
 RUN go get github.com/fabric8io/gobump
+
+# installing latest version of git
+RUN yum remove -y git
```

Comment:
Actually I initially tried this -> `RUN yum install -y wget make hg svn bzr gcc docker https://centos7.iuscommunity.org/ius-release.rpm git2u && \
  yum clean all`, but for some reason it was not installing git2u and raising the below error:
```
 ---> 0b7962480c44
Removing intermediate container 2365f467a122
Step 16 : RUN go get github.com/fabric8io/gobump
 ---> Running in 29af9b7490b8
go: missing Git command. See https://golang.org/s/gogetcmd
package github.com/fabric8io/gobump: exec: "git": executable file not found in $PATH
```
Hence was using removing and installing it later at the end.
But now I was checking and found that we can do it this way `RUN yum install -y wget make hg svn bzr gcc docker https://centos7.iuscommunity.org/ius-release.rpm && yum install -y git2u && \
  yum clean all`